### DISCUSSION TITLE
Easy access running timer

### DISCUSSION THREAD

**Discussion by SebastiaanKloos** [schmidfelix/timey#14] [Ideas] [answered]
It would be great to append the running timer to the projects page.

![CleanShot 2021-03-23 at 12 40 52](https://user-images.githubusercontent.com/6506510/112141567-b66fb080-8bd5-11eb-833f-f91eb19bac6f.png)

**Comment by schmidfelix** [comment, marked as answer]
Hey, 

another great suggestion! Will be included in the next release 🚀
